### DISCUSSION TITLE
How do I set the transaction isolation level?

### DISCUSSION THREAD

**Discussion by siddhsql** [TPC-Council/HammerDB#543] [Q&A] [answered]
How do I set the transaction isolation level while testing with HammerDB? I would like to know how to do it in the CLI

**Comment by sm-shaw** [comment, marked as answer]
Transaction isolation is a database setting.  For example, this is how to set it in MySQL
https://dev.mysql.com/doc/refman/8.0/en/server-system-variables.html#sysvar_transaction_isolation
Please consult the documentation for the database you are using.

**Reply by siddhsql** [reply]
Thanks. So does HammerDB not set the transaction isolation level? where can I see the stored procs it executes as part of the tests?

**Reply by sm-shaw** [reply]
No it doesn't set the transaction isolation level. You can see the stored procedures by looking in the stored procedures of the generated database or the HammerDB source on GitHub of course.
